README.md

```diff
@@ -4,6 +4,12 @@
 
 Applications to show how mikro-e [sensor clicks](http://www.mikroe.com/add-on-boards/sensors/) can be used with Ci40.
 
+###How to install applications of this package in OpenWrt's filesystem
```

Comment:
Just selecting in menuconfig it won't install. Mention that need to build openwrt with usual way, and load the image.